portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -9,16 +12,27 @@
 @WebServlet("/form-handler")
 public class FormHandlerServlet extends HttpServlet {
 
-  @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String name = request.getParameter("name");
+        String email = request.getParameter("email");
+        String phone = request.getParameter("phone");
+        String message = request.getParameter("message");
+
 
-    // Get the value entered in the form.
-    String textValue = request.getParameter("text-input");
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted name: " + name);
+        System.out.println("You submitted email: " + email);
+        System.out.println("You submitted phone: " + phone);
+        System.out.println("You submitted message: " + message);
 
-    // Print the value so you can see it in the server logs.
-    System.out.println("You submitted: " + textValue);
 
-    // Write the value to the response so the user can see it.
-    response.getWriter().println("You submitted: " + textValue);
-  }
+        // Write the value to the response so the user can see it.
+        response.getWriter().println("You submitted name: " + name);
```

Comment:
same here, use one response.getWriter() for this especially if this is temporary.